config.json

```diff
@@ -2,20 +2,20 @@
     "UseFoodPerAgentRatio": false,
     "FoodOnPlatform": 100,
     "FoodPerAgentRatio": 0,
-    "Team1AgtOne": 2,
-    "Team1AgtTwo": 2,
-    "Team2Agents": 2,
-    "Team3Agents": 2,
-    "Team4Agents": 2,
-    "Team5Agents": 2,
-    "Team6Agents": 2,
-    "Team7Agent1": 2,
-    "RandomAgents": 2,
+    "Team1AgtOne": 0,
+    "Team1AgtTwo": 0,
+    "Team2Agents": 0,
+    "Team3Agents": 0,
+    "Team4Agents": 0,
+    "Team5Agents": 0,
+    "Team6Agents": 10,
+    "Team7Agent1": 0,
+    "RandomAgents": 0,
     "AgentHP": 100,
     "AgentsPerFloor": 1,
     "TicksPerFloor": 10,
-    "SimDays": 8,
-    "ReshuffleDays": 1,
+    "SimDays": 16,
+    "ReshuffleDays": 4,
```

Comment:
I'd avoid changing anything outside of team6 - this is all infra stuff that will mess up PRs in future. Can u revert plz